StitchMDBKit/StitchMDBKit/LocalMongoDBService.swift

```diff
@@ -0,0 +1,168 @@
+import Foundation
+import MongoMobile
+import os
+import StitchCore
+import StitchCore_iOS
+
+/// Commands associated with reporting battery level
+/// to MongoMobile
+private enum BatteryLevelCommand: String {
+    case mongoCommand = "setBatteryLevel"
+    case batteryLevelLow = "low"
+    case batteryLevelNormal = "normal"
+}
+
+/// Current state of the UIDevice battery
+private enum BatteryState {
+    case low, normal
+}
+
+/// Commands associated with reporting memory level
+/// to MongoMobile
+private enum TrimMemoryCommand: String {
+    case mongoCommand = "trimMemory"
+    case aggressiveLevel = "aggressive"
+}
+
+/// Name of admin database to communicate commands to
+private let adminDatabaseName = "admin"
+
+/// Cached mongoClients associated to app ids
+private var localInstances = [String: MongoClient]()
```

Comment:
[nit] why is this a global rather than a static member of a singleton like it is in Java?

Reply:
The same could go for a lot of other stuff in this file.

Reply:
Because that's not a particularly Swifty way to do it. It's more expressive if, these static members are only used within this file, to have them at the file level.